pkg/event/eventDispatcher.go

```diff
@@ -54,7 +54,7 @@ func Dispatch(vwoInstance schema.VwoInstance, impression schema.Impression) {
 			message := fmt.Sprintf(constants.ErrorMessageImpressionFailed, vwoInstance.API, err)
 			utils.LogMessage(vwoInstance.Logger, constants.Error, eventDispatcher, message)
 		} else {
-			message := fmt.Sprintf(constants.InfoMessageImpressionSuccess, vwoInstance.API, "Normal", impression)
+			message := fmt.Sprintf(constants.InfoMessageImpressionSuccess, vwoInstance.API, "Normal", URL)
```

Comment:
is this right?

Reply:
Earlier we were printing the whole impression, in other SDK they have just logged the endpoint (URL)